langchain/output_parsers/structured.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import json
+from typing import List
+
+from pydantic import BaseModel
+
+from langchain.output_parsers.base import BaseOutputParser
+from langchain.output_parsers.format_instructions import STRUCTURED_FORMAT_INSTRUCTIONS
+
+line_template = '\t"{name}": {type}  // {description}'
```

Comment:
Should this have a `,` at the end of each line other than the last one? That would likely lead to a higher likelihood of getting valid json from the model

Reply:
i havent experienced any real issues with this so far

Reply:
i think because its defined as a schema, and not as an example, it works

Reply:
it might be because it's interpolating between typescript and json somehow. no need for a comma with typescript